### ISSUE TITLE
.swiftformat & xcode plugin do not work

### ISSUE THREAD

**Issue by oboje** [nicklockwood/SwiftFormat#1946]
> Alternatively, if you name the file .swiftformat and place it inside the project you are formatting, it will be picked up automatically.

my swiftformat xcode plugin ignores .swiftformat file? Should plugin pick up .swiftformat as well ?

**Comment by nicklockwood** [comment]
No, the Xcode Extension is sandboxed and cannot access the file syste
